src/services/BavService.ts

```diff
@@ -203,4 +204,54 @@ export class BavService {
 		}
 		return sessionId;
 	}
+
+	async getSessionByAuthorizationCode(code: string | undefined): Promise<ISessionItem | undefined> {
```

Comment:
I might be getting a bit nitpicky here but we have validated our code in the `isPayloadValid` function so is it safe to assume we will always get a string? 

Just thinking that there is really no point doing a DB call we know is going to fail, so if it is possible `code` could be undefined maybe we put a check in before we make it?

Reply:
removed undefined.